packages/storybook/stories/MessageThread/MessageThread.stories.tsx

```diff
@@ -339,13 +383,17 @@ const MessageThreadStory = (args): JSX.Element => {
       />
       {/* We need to use these two buttons to render more messages in the chat thread and showcase the "new message" button.
         Using storybook controls would trigger the whole story to do a fresh re-render, not just components inside the story. */}
-      <Stack horizontal horizontalAlign="space-between" tokens={{ childrenGap: '1rem' }}>
-        <PrimaryButton text="Send new message from others" onClick={onSendNewMessageFromOthers} />
-        <PrimaryButton text="Send new message" onClick={onSendNewMessage} />
-        <PrimaryButton text="Send new message with inline image" onClick={onSendNewMessageWithInlineImage} />
-        <PrimaryButton text="Send new system message" onClick={onSendNewSystemMessage} />
-        <PrimaryButton text="Send new custom message" onClick={onSendCustomMessage} />
-        <PrimaryButton text="Send new message with mention" onClick={onSendNewMessageWithMention} />
+      <Stack horizontal verticalAlign="end" horizontalAlign="center" tokens={{ childrenGap: '1rem' }}>
+        <Dropdown
+          style={{ width: '15rem' }}
+          label="Send to thread"
+          selectedKey={selectedMessageType.key}
+          options={dropdownMenuOptions}
+          onChange={(_, option) => {
+            setSelectedMessageType(option);
+          }}
+        />
+        <PrimaryButton text="Send" onClick={onClickHandler} />
```

Comment:
nitpick: maybe rename the onClickHandler
```suggestion
        <PrimaryButton text="Send" onClick={onSendMessageHandler} />
```